canopen/sdo/client.py

```diff
@@ -731,14 +736,38 @@ def _block_ack(self):
             raise SdoCommunicationError("Server did not respond with a "
                                         "block download response")
         if ackseq != self._blksize:
+            # Sequence error, try to retransmit
             self.sdo_client.abort(0x05040003)
```

Comment:
I don't quite remember the spec, is this abort code still needed?

Reply:
![image](https://user-images.githubusercontent.com/54152208/183304863-a80812da-860e-4cc4-b764-ba7beb43c4db.png)
This is taken from the spec and apparently an SDO abort is only necessary in case of a fatal error, so  I will remove it.
However it still worked with this abort message, so we probably have a bug on our side.

Reply:
No it's not very common for clients to send abort codes to the server, so some servers may just ignore them.

Reply:
Ok, good to know.